graphql/resolvers.go

```diff
@@ -63,6 +64,37 @@ func (r *hostResolver) Elapsed(ctx context.Context, obj *restModel.APIHost) (*ti
 	return obj.RunningTask.StartTime, nil
 }
 
+func (r *hostResolver) InstanceTags(ctx context.Context, obj *restModel.APIHost) ([]host.Tag, error) {
```

Comment:
it seems kind of unusual there is a separate resolver for a field that already exists in APIHost.

Reply:
Your right i'm not sure why i did that i just removed it. Thanks for catching that.